Dockerfile.release

```
@@ -1,4 +1,10 @@
-FROM registry.access.redhat.com/ubi8/ubi-minimal:8.7
+FROM --platform=linux/amd64 registry.access.redhat.com/ubi8/ubi-minimal:latest as build
```

```suggestion
FROM --platform=linux/${TARGETARCH} registry.access.redhat.com/ubi8/ubi-minimal:8.7 as build
```

- We should stick to a release UBI container and not use `:latest` tag
- TargetArch allows multi-arch containers